platform/ui/src/contextProviders/ViewportGridProvider.tsx

```diff
@@ -129,6 +129,11 @@ export function ViewportGridProvider({ children, service }) {
        * render partially in various ways causing exceptions.
        */
       case 'SET_DISPLAYSETS_FOR_VIEWPORTS': {
+        console.log(
```

Comment:
remove

Reply:
I shouldn't have added this file (removed it)